meta/def.rb

```diff
@@ -94,6 +110,7 @@
   mutation 'def foo(_unused = false); end'
   mutation 'def foo(_unused = true); raise; end'
   mutation 'def foo(_unused); end'
+  mutation 'def foo; _unused = true; end'
```

Comment:
Should the mutation just be `def foo; end`? What would be the point in emitting a variable prefixed with an underscore? 

BTW I'm not asking a rhetorical question, I don't know the answer, but I thought maybe it was worth discussing. There could be a good reason why we don't leave it out. It might be a situation where we all agree it should be left out, but we decide to do it later because it brings more value to commit this change now, and then work on optimizing the output in a future PR.

In general our approach is that we attempt to merge a PR if it brings an incremental improvement provided it meets code review and doesn't cause any metrics regressions. There are some changes that are blockers to getting PR merged, but if it's a case where adding it now vs. later makes no difference, then we'd usually defer it to get the improvement in now and then work on a follow-up PR making the additional change.